custom_components/kia_uvo/vehicle.py

```diff
@@ -66,32 +67,33 @@ async def async_update_data():
             update_method=async_update_data,
         )
 
-    async def refresh(self, interval: bool = False):
+    async def update(self, interval: bool = False):
         _LOGGER.debug(f"Vehicle:{self.__repr__()}")
         old_vehicle_status: dict = self.__repr__()
-        local_timezone = dt_util.UTC
-        event_time_local = dt_util.utcnow().astimezone(local_timezone)
-        if self.last_updated is None:
-            _LOGGER.debug(f"last updated is None!")
+        api_timezone = dt_util.UTC
+        event_time_api = dt_util.utcnow().astimezone(api_timezone)
```

Comment:
@cdnninja heads up this was a lot of the stability issues I was facing from the old code, the no scan window was checking against the api's time zone instead of the local time zone so half my day was off when it shouldn't have been

Reply:
Thanks for the heads up. I will fix this as well.